Core/src/IfcGeometryConverter/GeomUtils.cpp

```diff
@@ -150,10 +150,10 @@ void GeomUtils::extrude(
 	//  0-------face_loops[0]--------1
 
 	carve::geom::vector<3> normal_first_loop;
-	std::vector<std::vector<carve::geom2d::P2> >	face_loops;
-	for( std::vector<std::vector<carve::geom::vector<2> > >::const_iterator it_face_loops = face_loops_input.begin(); it_face_loops != face_loops_input.end(); ++it_face_loops )
+	std::vector<std::vector<carve::geom2d::P2>>	face_loops;
+	for( std::vector<std::vector<carve::geom::vector<2>>>::const_iterator it_face_loops = face_loops_input.begin(); it_face_loops != face_loops_input.end(); ++it_face_loops )
 	{
-		const std::vector<carve::geom::vector<2> >& loop = (*it_face_loops);
+		const std::vector<carve::geom::vector<2>>& loop = (*it_face_loops);

```

Comment:
Such for loops can be greatly simplified using the notation `for( auto element : array )` with corresponding names for element and array.

Reply:
Resolved: 1805f06